### DISCUSSION TITLE
Add a draggable scroll bar to the "Manage collections" tab, other helpful collection features

### DISCUSSION THREAD

**Discussion by domeosu** [ppy/osu#30736] [Ideas] [answered]
-I noticed that in the "Manage collections" tab there’s no way to quickly scroll down using a draggable scroll bar. It’d be super helpful to have one on the right side, considering that to create a new collection i have to manually scroll all the way down. Considering that, it would be greatly appreciated to have the "create a new collection" option to the top instead of having it all the way to the bottom, really helpful especially for people with a lot of collections.
-Have the search feature implemented for both the "manage collections" and collection drop down menu tabs similiar to the one already present in the beatmap selector page.
-When confirming the name change of a collection it's pretty annoying to be brought all the way to the top, please make the page current position not reset everytime a name change occurs in the "manage collections" menu.

**Comment by peppy** [comment]
> When confirming the name change of a collection it's pretty annoying to be brought all the way to the top, please make the page current position not reset everytime a name change occurs in the "manage collections" menu

Have you updated to the latest release? This should already be fixed.

**Reply by domeosu** [reply]
![image](https://github.com/user-attachments/assets/1036e2d5-048c-4815-ae58-ffd71c7d0f79)
i think i am, and it's still there

**Reply by peppy** [reply]
COuld you please take a video showing what you mean?

**Reply by domeosu** [reply]
https://github.com/user-attachments/assets/1f11db5c-cd89-4f9e-96b9-59d00bd4556f

**Reply by Joehuu** [reply]
Unfortunately, the fixes were not included in the current release. Will need to wait for the next one.

**Reply by peppy** [reply, marked as answer]
https://github.com/ppy/osu/pull/30743 for reference.

**Reply by domeosu** [reply]
> #30743 for reference.

could you also check out this? https://github.com/ppy/osu/discussions/30735
i know it can be tedious to implement but i think it's possible
